docs/src/api.md

```diff
@@ -0,0 +1,22 @@
+# AdvancedMH.jl
+
+Documentation for AdvancedMH.jl
+
+```@contents
+```
+
+## Structs
+```@docs
+StaticMH
+```
+
+## Functions
+
+```@docs
+sample
```

Comment:
I'm not sure if this should be added here (and if there's a docstring related to MH at all). It's defined in upstream packages.

Reply:
```suggestion
```